core/src/main/scala/com/carto/analyticstoolbox/spark/SpatialFilterPushdownOptimizations.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.carto.analyticstoolbox.spark
```

Comment:
Probably we need to move this file into the `com.carto.analyticstoolbox.spark.sql` package (to keep it consistent with Spark), and rules will move to `com.carto.analyticstoolbox.spark.sql.rules` in this case.